tools/README.md

```diff
@@ -19,13 +17,13 @@ from the given docker images.
 (Docker is required for using network configurations other than "local")
 
 ```
-yarn install
+npm install
 ```
 
 ### Usage
 
 ```
-yarn run launch --parachain moonbase-0.11.2
+npm run launch --parachain moonbase-0.11.2
```

Comment:
are you sure this works ? npm used to require `--` to pass parameters to the command

Reply:
It is not because of more than one parameter, it is to differentiate the npm parameters from the program (here `launch`) parameters